OpenCL_C.txt

```diff
@@ -7868,14 +7877,15 @@ semantics of the minimum requirements.
 The OpenCL C programming language implements the following additional
 built-in vector functions.
 We use the generic type name `gentype__n__` (or `gentype__m__`) to indicate the
```

Comment:
Btw is `__m__` explained somewhere?

Reply:
I don't think it is precisely defined, but I think it should have the same possible values of `__n__`, correct?

If so, I could change from:

> _n_ is 2, 4, 8, or 16.

to:

> _n_ or __m__ is 2, 4, 8, or 16.

It may be easier to make this change in a subsequent PR though.

Reply:
sure!